CorpusSearch/ClientApp/src/routes/Newbies.tsx

```diff
@@ -0,0 +1,45 @@
+import React, {useEffect, useState} from "react"
+function CorpusLink({ident="",name=""}): JSX.Element {
+    if ( name.slice(0,2) != "🎥"){
+        name = "📖" + name
+    }
+    return <div>
+        <a href={"docs/" + ident}>{name}</a>
+    </div>
+}
+export const Newbies = () => {
+type DocType =  {name:string, ident:string, uploaded:string}
+const [newDocs, setNewDocs  ] 
+    = useState<DocType[]>([])
+useEffect(()=>  {
+    const fetchNewDocs = async () => {
+        await fetch("api/metadata/latest")
+            .then((res)=> {
+                if (!res.ok) {
+                    console.warn(`metadata/latest returned status: ${res.status}`)
+                }
+                return res.json()
+            }).then((json: DocType[])=> {
+                setNewDocs(json)
+        })
+    }
+    fetchNewDocs().catch((e:Error)=> console.warn(`fetchNewDocs returned error:" ${e.message}`))
+},[]
+    )
+    const len = newDocs.length
+    if (!len) return <></>
+    const newarray: JSX.Element[] = []
+    for ( let i=0; i < len; i++) {
+        const {name,ident} = newDocs[i]
+        const link = <CorpusLink key={name} name={name} ident={ident} />
+        newarray.push(link)
+    }
+    return (<div className="Newbies">
+        <header className="Newbies-header">
+        </header>
+        <div>New and Featured Transcriptions:<br/>
+            <div>{newarray}</div>
```

Comment:
I'd consider an unordered list:` <ul><li>item1</li><li>item2</li></ul>`

Reply:
What does this get us?

Reply:
Bullet points are better-spaced by default

Reply:
Also how to combine that with your other comment about the map?

Reply:
Something like:

```tsx
<ul>
    {newDocs.map(doc => <li key={doc.name}><CorpusLink name={doc.name} ident={doc.ident} /></li>)}
</ul>
```

Reply:
Okay, just pushed a bunch of other changes, can try this next I guess